src/status_im/subs/wallet/wallet.cljs

```diff
@@ -165,12 +165,17 @@
  (fn [accounts]
    (filter :watch-only? accounts)))
 
+(defn to-lower-case
+  [coll]
+  (map string/lower-case coll))
+
 (rf/reg-sub
  :wallet/addresses
  :<- [:wallet]
  :-> #(->> %
            :accounts
            keys
+           to-lower-case
```

Comment:
maybe you can create a separate subscription for this? `wallet/lowercase-addresses` based on this one, so we leverage caching